www/src/pages/insights.tsx

```diff
@@ -365,7 +365,7 @@ export default function InsightsPage({ pageContext: { tab } }: { pageContext: {
 
   const handleOnSelect = (value: string) => {
     if (value !== tab) {
-      global.analytics.track('Usage Insights', { tab: value });
+      global.analytics.track(`openedx.paragon.docs.insights.tabs.${value.toLowerCase().trim()}.clicked`, { tab: value });
```

Comment:
Same here, we probably don't need to send additional data (`{ tab: value }`) here because event name already indicates which tab was clicked (e.g., sending `openedx.paragon.docs.insights.tabs.summary.clicked` event is enough to know that `Summary` tab was clicked, no need to also send `{ tab: 'summary' }`)

Reply:
Corrected, thanks